client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/style.scss

```
@@ -312,48 +241,75 @@ h2.action-panel__title {
 	}
 }
 
-.email-providers-stacked-comparison__main.main {
-	margin: 12px auto;
+.card.action-panel.promo-card.email-providers-stacked-comparison__provider-card {
+	display: flex;
+	flex-direction: column;
 
-	@include break-mobile {
+	@include breakpoint-deprecated( '>660px' ) {
 		flex-direction: row;
 	}
 
-	.card.action-panel.promo-card.email-providers-stacked-comparison__provider-card {
+	.action-panel__figure {
 		display: flex;
-		flex-direction: column;
+		margin-top: 3px;
+		flex-direction: row;
+		text-align: left;
+		margin-bottom: 0px;
+		.professional-email-card__logo {
+			min-width: 48px;
+		}
 
-		.action-panel__figure {
-			display: flex;
-			margin-top: 3px;
-			flex-direction: row;
+		@include breakpoint-deprecated( '<660px' ) {
 			max-width: 100%;
-			text-align: left;
-			margin-bottom: 0px;
-			.professional-email-card__logo {
-				min-width: 48px;
-			}
+		}
 
-			@include break-mobile {
-				order: 1;
+		@include break-mobile {
+			order: 1;
 
-				&.align-left {
-					order: -1;
-				}
+			&.align-left {
+				order: -1;
 			}
 		}
 	}
 }
 
-.email-provider-stacked-features__toggle-button {
-	overflow: visible;
-	white-space: nowrap;
-	cursor: pointer;
-	display: flex;
-	justify-content: space-between;
-	margin-top: 8px;
-	width: 100%;
-	.gridicon {
-		margin-left: 5px;
+.email-provider-stacked-card__provider-form > * .form-text-input-with-affixes {
+	flex-direction: column;
+
+	.form-text-input {
+		@include breakpoint-deprecated( '<480px' ) {
+			max-width: 100%;
+		}
+	}
+
+	.form-text-input-with-affixes__suffix {
+		border-top-width: 0;
+		border-left-width: 1px;
+		border-left-style: solid;
+		border-left-color: var( --color-neutral-10 );
+		display:inline-block;
+		overflow: hidden;
+		text-overflow:ellipsis;
+		white-space:nowrap;
```

Nit: please add spaces after the colon.
```suggestion
		display: inline-block;
		overflow: hidden;
		text-overflow: ellipsis;
		white-space: nowrap;
```